src/server/game/Chat/ChatCommands/ChatCommand.cpp

```diff
@@ -345,9 +344,7 @@ namespace Acore::Impl::ChatCommands
     }
 
     if (!sScriptMgr->CanExecuteCommand(handler, cmdStr))
```

Comment:
Why are we having this twice here? 👀

Reply:
I don't know, it was originally there so I just left it. It can probably be removed.